Dockerfile

```diff
@@ -1,4 +1,22 @@
-FROM ruby:2.7-slim-buster
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# GTT Builder
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+FROM node:16-bullseye-slim as gtt-builder
+
+WORKDIR /app
+
+COPY plugins/redmine_gtt/ ./redmine_gtt/
+
+RUN apt update; \
+    apt install -y git; \
+    cd redmine_gtt; \
+    yarn; \
+    yarn webpack
```

Comment:
This is a slight thing, but in `redmine_gtt` plugin's `README.md`, we use `npx webpack`, but is it better to change `yarn webpack` in the `README.md` side ?
https://github.com/gtt-project/redmine_gtt#installation